backend/app/users/routers.py

```diff
@@ -42,8 +43,20 @@ async def create_user(
     """
     Create user endpoint.
     """
-
     try:
```

Comment:
This has become a very large try and except block with a tonne of things in there. Can we put the try and except around the ones actually expected to throw the `UserAlreadyExistsError` error?

Reply:
Done